dataactbroker/README.md

```diff
@@ -240,65 +240,74 @@ Example Output:
 ```
 
 #### POST "/v1/submit_files/"
```

Comment:
Remember that this documentation needs to also support the functionality currently on the front end.

Reply:
^ Don't forget that note.